### PR TITLE
HomeWork2

### PR DESCRIPTION
**Pull request by AlexisKozlov**
1. Task https://www.figma.com/file/16bdlWFoZXXIpyyZmwy9JU/Logoipsum?node-id=1%3A2
2. 
image
 
 3. https://alexiskozlov.github.io/tms/
P.s. Не вышло сделать по позиционированию, пока не знаю как.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    display: flex;
+    justify-content: space-around;
+}
+
+ .logo {
+ padding-top: 41px;   
+ /* margin-left: 184px;  */
+ /* margin-right: 173px; */
+ /* padding-right: 123px; */
+}
```
Comment (shuliakleonid) [inline style.css:14]: comments in your code is bad practice

File: style.css [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    margin-top: 100px;
+    /* margin-left: auto;
+    margin-right: auto;
+    width: em; */
+}
+
+/* .title__grow{
+  margin-top: 92px;  
+  margin-left: 184px;
+} */
```
Comment (shuliakleonid) [inline style.css]: if you need comments that's all right

**Review by shuliakleonid — CHANGES_REQUESTED** [review]

**Comment by shuliakleonid** [comment]
## Good work!

File: style.css (hunk 1 above)
Comment (AlexisKozlov) [reply style.css:14]: > FIxed

**Review by shuliakleonid — APPROVED** [review]

**Review by egkirpichev — APPROVED** [review]

**Review by KseniaKotova — APPROVED** [review]
Good going!

**Pull request merged**